Replace “status” by “post” in strings

ariasuni opened the pull request:
«Toot» has been replaced by «post» in strings, but there was a few «status» left as well. Now everything is called a post in the UI.

For reference, the same change in Mastodon: https://github.com/mastodon/mastodon/pull/16271

connyduck commented:
Oops I forgot those in https://github.com/tuskyapp/Tusky/pull/2396
Can you also rename all the string names like I did there, please?

ariasuni commented:
I think it’s good now? Since I used search and replace, it changed a lot of files (also changed a view ID), but I double-checked everything. Also fixed a «postss» into «posts» (thanks to @gunchleoc for the comment on Weblate)

connyduck reviewed (approved):
Thank you!
